Rework sprites list to make button sticky

ClementPasteau opened the pull request:
Remove the button from the Scrollable list.

The mobile view is a bit crowded, but this was similar before.
Maybe we could gather buttons so they are displayed 2 per row (collision masks + points for instance)

![Capture d’écran 2023-03-10 à 17 50 33](https://user-images.githubusercontent.com/4895034/224376131-60461963-65c6-4aaa-8c9e-a518e6797037.png)
![Capture d’écran 2023-03-10 à 17 53 15](https://user-images.githubusercontent.com/4895034/224376137-73f426c4-0a8f-4c04-afdd-6206b31a603c.png)
![Capture d’écran 2023-03-10 à 17 53 26](https://user-images.githubusercontent.com/4895034/224376139-254c8389-0c06-4643-b012-dfe3513ae681.png)

AlexandreSi commented:
I noticed that reordering doesn't trigger the "Are you sure you want to cancel?" dialog. It might be a nice addition to this.

AlexandreSi commented:
Furthermore, I noticed that you can drag an image that is alone in an animation. Might be worth removing this (could be cumbersome on mobile if you accidentally put your thumb on it?)

AlexandreSi commented on newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpritesList.js:
```diff
@@ -203,203 +157,246 @@ type Props = {|
-    );
-  }
-}
+        <Column>
```
Add noMargin here imho

ClementPasteau replied:
done

AlexandreSi commented:
Finally, on some low res mobile, you can't see the buttons on the left:

<img width="343" alt="image" src="https://user-images.githubusercontent.com/32449369/224392623-48006673-114f-4c4d-ac91-de719da3da83.png">

So with longer translations, it will happen for sure.

ClementPasteau commented:
> I noticed that reordering doesn't trigger the "Are you sure you want to cancel?" dialog. It might be a nice addition to this.

Good catch - fixed.

> Furthermore, I noticed that you can drag an image that is alone in an animation. Might be worth removing this (could be cumbersome on mobile if you accidentally put your thumb on it?)

Good catch too! fixed

> Finally, on some low res mobile, you can't see the buttons on the left (Edit and Preview):

Yep definitely cumbersome on mobile. I've reworked briefly the layout so that it stack in a column instead of disappearing

See screenshots above

AlexandreSi commented:
On mobile, the spacing is weird:
<img width="295" alt="image" src="https://user-images.githubusercontent.com/32449369/225059890-3859042b-46dc-4234-b971-e6652aa2c524.png">

There's a lot of space between Preview and Loop. But i'm not sure how to do it better. With a flex wrap somewhere?

AlexandreSi reviewed (approved):
LGTM after the spacing weird look

ClementPasteau commented:
good catch, I believe it's better now:
![image](https://user-images.githubusercontent.com/4895034/225063986-1f37778e-4bca-4d7d-9ae4-be8817d8391a.png)
